runtime/databricks/automl_runtime/forecast/pmdarima/model.py

```diff
@@ -145,7 +145,7 @@ def predict(self, context: mlflow.pyfunc.model.PythonModelContext, model_input:
 
     def _predict_impl(self, input_df: pd.DataFrame) -> pd.DataFrame:
         df = input_df.rename(columns={self._time_col: "ds"})
-        df["ds"] = pd.to_datetime(df["ds"])
+        df["ds"] = pd.to_datetime(df["ds"], infer_datetime_format=True, errors="coerce")
```

Comment:
Actually, on second thought, I think that we don't want `errors="coerce"` during inference so that the error is surfaced to the user. Let's remove that but keep the `infer_datetime_format=True`